server/chat-plugins/tcgtabletop.ts

```diff
@@ -1,34 +1,35 @@
 /**
 * TCG & Tabletop: Yugioh wiki plugin
-* This is a command that allows users to search the yugioh wiki for cards. It will display the closest match with a given query, or a separate message if there isn't anything found.
+* This is a command that allows users to search the yugioh wiki for cards.
+* It will display the closest match with a given query, or a separate message if there isn't anything found.
 * By bumbadadabum with help from ascriptmaster, codelegend and the PS development team.
 */
 
-'use strict';
 
-const https = require('https');
-const querystring = require('querystring');
+import * as https from 'https';
+import * as querystring from 'querystring';
+
 
 const SEARCH_PATH = '/api/v1/Search/List/';
 const DETAILS_PATH = '/api/v1/Articles/Details/';
 
-async function getFandom(site, pathName, search) {
+async function getFandom(site: string, pathName: string, search: AnyObject) {
 	const reqOpts = {
 		hostname: `${site}.fandom.com`,
 		method: 'GET',
-		path: `${pathName}?${querystring.stringify(search)}`,
+		path: pathName + '?' + querystring.stringify(search),
```

Comment:
```suggestion
		path: `${pathName}?${querystring.stringify(search)}`,
```

Can you revert this? I think I prefer how it was before.